rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

```diff
@@ -78,13 +112,12 @@ After deinstallation or reconfiguration, a bot is still shown in the *Bots* view
 . In the *Bots* view of the *Bot Management* module, click *Remove* (image:delete-icon.png[trash symbol,1.5%,1.5%]) in the row of the bot to remove.
 . Confirm the removal.
 
-If you accidentally remove a bot that is still being executed, the bot will shut down and the API Key is deleted on the bot's computer. You must uninstall and reinstall the bot with a new API Key. In this case, the licenses, name, description, assigned process applications and service times, and tasks will be lost and have to be assigned again.
+If you accidentally remove a bot that is still being executed, the bot shuts down and the API Key is deleted on the bot's computer. You must uninstall and reinstall the bot with a new API Key. In this case, the licenses, name, description, assigned process applications and service times, and tasks are lost and have to be assigned again.
 
 == See Also
 
-* xref:rpa-bot::hardware-software-requirements.adoc[Hardware and Software Requirements]
-* xref:botmanagement-overview.adoc[Bot Management]
-* xref::botmanagement-support.adoc[Supporting Bots]
-* xref::botmanagement-know.adoc[Getting to Know Bots]
-//* xref::botmanagement-manage.adoc[Managing Bots]
-* xref::botmanagement-troubleshoot.adoc[Troubleshooting Bots]
+* xref:rpa-home::automation-security-securesessions.adoc[Secure Sessions]
```

Comment:
If the headings in the H1 are the same, remove the hard-coded links to autogenerate (and reduce maintenance).

Reply:
I agree, but would like to include the product name in the link text when I link to docs outside the current second navigation level.

Reply:
no, but consistency gives me security :)
I will revisit taht with the renewal of the RPA content structure project...